build.sbt

```diff
@@ -335,8 +346,16 @@ lazy val testkit = crossProject.crossType(CrossType.Pure)
   .settings(moduleName := "cats-testkit")
   .settings(catsSettings)
   .settings(disciplineDependencies)
-  .settings(
-    libraryDependencies += "org.scalatest" %%% "scalatest" % scalaTestVersion)
+  // 2.13.0-M3 workaround
+  //.settings(libraryDependencies += "org.scalatest" %%% "scalatest" % scalaTestVersion)
+  .settings(libraryDependencies += {
+    CrossVersion.partialVersion(scalaVersion.value) match {
+      case Some((2, 13)) =>
+        "org.scalatest" %%% "scalatest" % scalatest_2_13
+      case _ =>
+       "org.scalatest" %%% "scalatest" % scalaTestVersion
```

Comment:
This logic is duplicated in two places. Should we create a `def scalatestVersion(scalaVersion: String): String` or something?

Reply:
They are not quite the same, the first has a the `test` scope. A "real" function looks more like [this](https://github.com/typelevel/sbt-catalysts/blob/master/src/main/scala/sbtcatalysts/CatalystsPlugin.scala#L108-L123) or [as in catalysts](https://github.com/typelevel/catalysts/blob/master/build.sbt#L298-L327). So I opted for a comment instead  i,e `// 2.13.0-M3 workaround`

So as this is commented as an interim workaround, is this OK for now?

Reply:
I can also add a link to the comment describing this interim workaround - https://github.com/scalatest/scalatest/issues/1321#issuecomment-365140680

Reply:
@BennyHill I opened a PR against your repo with the workaround that I was thinking might clean things up a little bit. Feel free to merge if you like it: https://github.com/BennyHill/cats/pull/1